src/grt/src/fastroute/include/FastRoute.h

```diff
@@ -150,9 +150,14 @@ class FastRouteCore
   int totalOverflow() const { return total_overflow_; }
   bool has2Doverflow() const { return has_2D_overflow_; }
   void updateDbCongestion();
+  std::vector<std::string> getNetsInCongestedEdge(int x, int y, bool vertical);
   void getCongestionGrid(
-      std::vector<std::pair<GSegment, TileCongestion>>& congestionGridV,
-      std::vector<std::pair<GSegment, TileCongestion>>& congestionGridH);
+      std::vector<
+          std::tuple<GSegment, TileCongestion, std::vector<std::string>>>&
+          congestionGridV,
```

Comment:
a struct rather than a tuple wold make this more compact and the fields more self-documenting